boards/arm/nucleo_wba52cg/doc/nucleo_wba52cg.rst

```diff
@@ -220,12 +220,31 @@ Default settings are 115200 8N1.
 Programming and Debugging
 *************************
 
+Nucleo WBA52CG board includes an ST-LINK/V3 embedded debug tool interface.
+It could be used for flash and debug using either OpenOCD or STM32Cube ecosystem tools.
```

Comment:
"... for flashing and debugging using either OpenOCD or the STM32Cube ecosystem tools."